Source/Core/VideoBackends/Software/Clipper.cpp

```diff
@@ -489,13 +489,15 @@ bool CullTest(const OutputVertexData* v0, const OutputVertexData* v1, const Outp
 
   backface = normalZDir <= 0.0f;
 
-  if ((bpmem.genMode.cullmode & 1) && !backface)  // cull frontfacing
+  if ((bpmem.genMode.cullmode == CullMode::Back || bpmem.genMode.cullmode == CullMode::All) &&
```

Comment:
These enum choices are odd with the comment.  Just checking to make sure that's intentional?

Reply:
The mask is correct (1 is back, 2 is front, 3 is all), but the behavior seems odd to me.  YAGCD says that the bit is for what should be rejected (REJECT_EN in GEN_MODE near [§5.11.1](http://hitmen.c02.at/files/yagcd/yagcd/chap5.html#sec5.11.1)).  It's possible that the software renderer's definition of backface is reversed from the hardware renderer's, which was canceled out here.  (The changes needed for #9350 were less than I expected at least).

Reply:
I've added a TODO comment for this; it might be an inconsistency with the software renderer but probably shouldn't be addressed here.